hack/make-rules/update/generated.sh

```diff
@@ -17,7 +17,7 @@
 set -o errexit -o nounset -o pipefail
 
 # cd to the repo root and setup go
-REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../.." &> /dev/null && pwd -P)"
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../.." &>/dev/null && pwd -P)"
```

Comment:
Nit: The space here is insignificant, right?

Reply:
Yeah, this is my editor automatically formatting the file.